engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/CachedConfluentSchemaRegistryClientFactory.scala

```diff
@@ -0,0 +1,31 @@
+package pl.touk.nussknacker.engine.avro.schemaregistry.confluent.client
+
+import org.apache.avro.Schema
+import pl.touk.nussknacker.engine.kafka.KafkaConfig
+import pl.touk.nussknacker.engine.util.cache.DefaultCache
+
+import scala.concurrent.duration.Duration
+
+class CachedConfluentSchemaRegistryClientFactory(maximumSize: Long, latestSchemaTtl: Option[Duration], expireAfterAccess: Option[Duration])
+  extends ConfluentSchemaRegistryClientFactory {
+
+  //Cache engine is shared by many of CachedConfluentSchemaRegistryClient
```

Comment:
let's add "the class is Serializable (e.g. during Flink process creation) and cache is not shared between serialized instances" so that no one is mistaken

Reply:
Done